UIAutomationTests/TestsAuthenticationUsernamePassword.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+using System.IO;
+using Xunit;
+
+namespace UIAutomationTests
+{
+    //https://xunit.net/docs/running-tests-in-parallel
+    //We don't want to run them in parallel (since we're doing automation UI testing) so we add the tests to the same test collection
+    [Collection("VisualStudioUIAutomationTestCollection")]
+    public class TestsAuthenticationUsernamePassword
+    {
+        UIAutomationTestFixture testFixture;
+
+        public TestsAuthenticationUsernamePassword(UIAutomationTestFixture tf)
+        {
+            testFixture = tf;
+        }
+
+        //2 tests here will be sufficient for now since we just want to see if we can authenticate and fetch data with odbc.datasource and odbc.query
+        [Fact]
+        public void OdbcQueryAW()
```

Comment:
I think it would be nice to express the test case in the test name.
For example, maybe we can change this name to "AuthenticateWith....."

Same with the other tests.

Reply:
Fully agree, the test names could be better. Will improve this in the future.